tests/restarting/from_7.1.0/VersionVectorDisableRestart-1.toml

```diff
@@ -0,0 +1,42 @@
+[[knobs]]
+enable_version_vector = true
+enable_version_vector_tlog_unicast = true
+
```

Comment:
"PROXY_USE_RESOLVER_PRIVATE_MUTATIONS" is not enabled here - is that intentional?

Reply:
Ok, you can ignore my above comment here - you still need to disable it when you disable version vector (to make this test somewhat realistic)?